awx_collection/tests/integration/targets/tower_inventory_source/tasks/main.yml

```diff
@@ -41,13 +41,12 @@
     that:
       - "result is changed"
 
-- name: Delete the inventory source with an invalid cred, source_project, sourece_script specified
+- name: Delete the inventory source with an invalid cred and source_project pecified
```

Comment:
pecified -> specified

Reply:
will merge if you push that change

Reply:
thank you for the review and sorry for that. I have fixed it now.